XamarinCommunityToolkit/Views/AvatarView/AvatarView.shared.cs

```diff
@@ -1,11 +1,19 @@
-﻿using Xamarin.Forms;
+﻿using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Xamarin.Forms;
 using static System.Math;
 
 namespace Xamarin.CommunityToolkit.UI.Views
 {
 	public class AvatarView : BaseTemplatedView<Frame>
 	{
 		const string emptyText = "X";
+		static readonly IImageSourceValidator imageSourceValidator;
+
+		static AvatarView() => imageSourceValidator = new ImageSourceValidator();
```

Comment:
should we allow users to implement there own IImageSourceValidator like we do for IColorScheme?

Reply:
I think no, at least for now